components/package.json

```diff
@@ -1,12 +1,29 @@
 {
-	"version": "0.1.0",
-	"description": "",
-	"main": "index.js",
+	"name": "components",
+	"private": true,
+	"version": "0.0.0",
 	"type": "module",
```

Comment:
this file is missing the package's entrypoint ("main" property), and that's why the imports are wonky

Reply:
never mind, I was looking at an outdated version of this file